tensorflow/compiler/xla/stream_executor/cuda/cuda_blas_lt.cc

```diff
@@ -23,12 +23,18 @@ limitations under the License.
 #include <utility>
 #include <vector>
 
+#if GOOGLE_CUDA
 #include "third_party/gpus/cuda/include/cublasLt.h"
 #include "third_party/gpus/cuda/include/cublas_v2.h"
+#endif
 #include "tensorflow/compiler/xla/status_macros.h"
 #include "tensorflow/compiler/xla/stream_executor/blas.h"
+#if GOOGLE_CUDA
```

Comment:
I suspect upstream won't like that we're adding these #if's throughout these cuda files, but having copies (rocm_blas_lt.cc) that are 90% the same is worse.